cli/parser/parser.go

```diff
@@ -683,6 +683,8 @@ func runBazelHelpWithCache() (string, error) {
 		// Make sure this server doesn't stick around for long.
 		"--max_idle_secs=10",
 		"help",
+		// Don't print any debug information
+		"--logging=6",
```

Comment:
Interesting, didn't know about this `logging` flag. Would it make sense to make `6` a named constant?

Reply:
(Seems like the levels don't have specific meanings, so maybe `maxLoggingLevel` could be a good name?)